### ISSUE TITLE
Missing Python 3.12 wheels

### ISSUE THREAD

**Issue by catcombo** [wilsonzlin/minify-html#160]
Hello and thank you for the very useful package!

I noticed that `minify-html` doesn't have builds for Python 3.12. It tries to build it locally but without a success.

```
Backend subprocess exited when trying to invoke build_wheel
  
  Running `maturin pep517 build-wheel -i /tmp/tmpmi2d2gaa/.venv/bin/python --compatibility off`
  💥 maturin failed
    Caused by: Cargo metadata failed. Do you have cargo in your PATH?
    Caused by: No such file or directory (os error 2)
  Error: command ['maturin', 'pep517', 'build-wheel', '-i', '/tmp/tmpmi2d2gaa/.venv/bin/python', '--compatibility', 'off'] returned non-zero exit status 1

  at ~/.local/share/pypoetry/venv/lib/python3.12/site-packages/poetry/installation/chef.py:147 in _prepare
      143│ 
      144│                 error = ChefBuildError("\n\n".join(message_parts))
      145│ 
      146│             if error is not None:
    → 147│                 raise error from None
      148│ 
      149│             return path
      150│ 
      151│     def _prepare_sdist(self, archive: Path, destination: Path | None = None) -> Path:

Note: This error originates from the build backend, and is likely not a problem with poetry but with minify-html (0.11.1) not supporting PEP 517 builds. You can verify this by running 'pip wheel --use-pep517 "minify-html (==0.11.1)"'.
```

Could you please provide builds for the latest Python version? Thank you in advance!

**Comment by zyv** [comment]
I have built a wheel locally on my machine:

https://github.com/moneymeets/minify-html/releases/tag/0.11.1-python3.12

@catcombo @IamAbbey @haidarCheaito 

You can use the link in `pyproject.toml` until this issue is fixed.

**Comment by wilsonzlin** [comment]
Thanks for raising, version `0.11.2` has been released which has builds for Python 3.12.

**Comment by zyv** [comment]
Thank you!
